includes/class-wcpdf-main.php

```diff
@@ -42,8 +42,10 @@ public function __construct() {
 		add_action( 'wp_scheduled_delete', array( $this, 'schedule_temporary_files_cleanup' ) );
 
 		// remove private data
-		add_action( 'woocommerce_privacy_remove_order_personal_data_meta', array( $this, 'remove_order_personal_data_meta' ), 10, 1 );
-		add_action( 'woocommerce_privacy_remove_order_personal_data', array( $this, 'remove_order_personal_data' ), 10, 1 );
+		if ( true === apply_filters( 'wpo_wcpdf_remove_order_personal_data_meta', true ) ) {
```

Comment:
Maybe like this is enough? I have renamed the filter, makes more sense to me like this.

```php
if ( apply_filters(  'wpo_wcpdf_remove_order_personal_data', true ) ) {
```

Reply:
That makes sense. I followed the way we check this in other places, but I like more your suggestion.